### PR TITLE
Add linter for rpmspec

### PR DESCRIPTION
**Pull request by cryptomilk**
This adds a linter for RPM spec files.

I miss a simple `make test` to quickly run the tests ...

### CODE REVIEW

File: lua/lint/linters/rpmspec.lua [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  ignore_exitcode = true,
+  parser = function(output, bufnr)
+    local diagnostics = {}
+
+    for _, parser in ipairs(parsers) do
+      local result = parser(output, bufnr)
+
+      for _, diagnostic in ipairs(result) do
+        table.insert(diagnostics, diagnostic)
+      end
```
Comment (mfussenegger) [inline lua/lint/linters/rpmspec.lua:36]:
```suggestion
      vim.list_extend(diagnostics, result)
```

Comment (cryptomilk) [reply lua/lint/linters/rpmspec.lua]: I've changed it. Thanks.

**Comment by mfussenegger** [comment]
Added a fixup to fix the tests. Just to be sure: The reported line numbers are 1-based right? (diagnostics lnum is 0 based)

**Comment by cryptomilk** [comment]
> Added a fixup to fix the tests. Just to be sure: The reported line numbers are 1-based right? (diagnostics lnum is 0 based)

The reported line numbers by rpmspec are 1-based.

**Comment by cryptomilk** [comment]
It isn't really clear from the README.md how to run the tests. Just using `busted tests/` didn't really work. I'm using `mini.test` for my projects. You can just add it as a subproject and everyone can run the tests, all you need is nvim.

The fixup commit looks fine for me.

**Pull request merged**